### PR TITLE
feat(DatePicker): support multiple api

### PR DESCRIPTION
**Pull request by HaixingOoO**
- https://github.com/Tencent/tdesign-vue/issues/3341

### CODE REVIEW

File: packages/products/tdesign-react/src/date-picker/date-picker.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ format | String | 'YYYY-MM-DD' | 仅用于格式化日期显示的格式，不
 inputProps | Object | - | 透传给输入框（Input）组件的参数。TS 类型：`InputProps`，[Input API Documents](./input?tab=api)。[详细类型定义](https://github.com/Tencent/tdesign-react/blob/develop/src/date-picker/type.ts) | N
 label | TNode | - | 左侧文本。TS 类型：`string \| TNode`。[通用类型定义](https://github.com/Tencent/tdesign-react/blob/develop/src/common.ts) | N
 mode | String | date | 选择器模式。可选项：year/quarter/month/week/date | N
+multiple | Boolean | false | 支持多选日期，但不支持与rangepicker、showTime、allowInput。TS 类型：`boolean` | N
```
Comment (uyarn) [inline packages/products/tdesign-react/src/date-picker/date-picker.md:23]:
```suggestion
multiple | Boolean | false | 支持多选日期，但不支持在range-picker中，或与enableTimePicker、allowInput 一起使用。TS 类型：`boolean` | N
```

File: packages/products/tdesign-react/src/date-picker/date-picker.en-US.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ format | String | 'YYYY-MM-DD' | \- | N
 inputProps | Object | - | Typescript：`InputProps`，[Input API Documents](./input?tab=api)。[see more ts definition](https://github.com/Tencent/tdesign-react/blob/develop/src/date-picker/type.ts) | N
 label | TNode | - | Typescript：`string \| TNode`。[see more ts definition](https://github.com/Tencent/tdesign-react/blob/develop/src/common.ts) | N
 mode | String | date | options: year/quarter/month/week/date | N
+multiple | Boolean | false | support multiple date，but not support rangepicker、showTime、allowInput。Typescript：`boolean` | N
```
Comment (uyarn) [inline packages/products/tdesign-react/src/date-picker/date-picker.en-US.md:23]:
```suggestion
multiple | Boolean | false | support multiple date，but not support being use together with range-picker、enableTimePicker and allowInput。Typescript：`boolean` | N
```

**Review by liweijie0812 — APPROVED** [review]

**Comment by HaixingOoO** [comment]
@uyarn review liweijie0812 update docs了

**Review by uyarn — APPROVED** [review]

**Pull request merged**